rules/false_positives/k8s_dashboard.yara

```diff
@@ -0,0 +1,16 @@
+rule mode_php_js : override {
+  meta:
+    description = "mode-php.js, mode-php_laravel_blade.js"
+    original_severity = "critical"
+    php_executor = "high"
+  strings:
+    $ace_define = "ace.define"
+    $ace_lib = "ace/lib"
+    $ace_mode = "ace/mode"
+    $ace_require = "ace.require"
+    $mode_php_laravel_blade = "ace/mode/php_laravel_blade"
+    $php_worker = "ace/mode/php_worker"
+    $php_worker2 = "PhpWorker"
+  condition:
+    all of ($ace*) and ($mode_php_laravel_blade or $php_worker) and $php_worker2
```

Comment:
I think this query can be simplified to something like "4 of 5" strings.

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).